### ISSUE TITLE
custom Header/Cookie

### ISSUE THREAD

**Issue by s0i37** [Nekmo/dirhunt#84]
Hello.
Does it support to specify cookie or header token for crawling authenticated content?

**Comment by Nekmo** [comment]
Hi @s0i37 ,

Currently it is not possible to add custom headers, but it is a nice feature to add to Dirhunt. Thanks :)

**Comment by s0i37** [comment]
https://github.com/Nekmo/dirhunt/pull/85

**Comment by s0i37** [comment]
I've fixed

**Comment by Nekmo** [comment]
Fixed in v0.8.0 (currently in the development branch). To test this new feature:

```
$ pip install -U https://github.com/Nekmo/dirhunt/archive/develop.zip
```
